.travis.yml

```diff
@@ -33,6 +33,9 @@ script:
 after_success:
   - bash <(curl -s https://codecov.io/bash)
 
+after_failure:
+  - ".travis/after_failure.sh"
```

Comment:
No quotes needed.

Reply:
Good eyes:), Thanks Grant!
I just pushed the fix for it